{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-18T19:56:00.000+0000",
      "latitude": 51.533390295338485,
      "estimatedRestoration": "2022-06-19T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0269'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0269",
      "reference": "JV0269",
      "maxLongitude": -0.3628,
      "engineerOnSiteTime": "2022-06-18T20:46:00.000+0000",
      "minLatitude": 51.5324,
      "name": "UB6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB6 9TX",
        "UB6 9TY",
        "UB6 9TZ",
        "UB6 9UB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3676,
              51.5348
            ],
            [
              -0.3628,
              51.5348
            ],
            [
              -0.3628,
              51.5324
            ],
            [
              -0.3676,
              51.5324
            ],
            [
              -0.3676,
              51.5348
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-18T21:00:00.000+0000",
      "UUID": "2a445680-049d-45a0-9429-af63564242bd",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.3676,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5348,
      "longitude": -0.3649479768119774,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-18T20:58:00.000+0000",
      "latitude": 51.37403497457744,
      "estimatedRestoration": "2022-06-19T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0303'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0303",
      "reference": "JV0303",
      "maxLongitude": -0.8573,
      "minLatitude": 51.3682,
      "name": "RG40 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG40 4HR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8711,
              51.3757
            ],
            [
              -0.8573,
              51.3757
            ],
            [
              -0.8573,
              51.3682
            ],
            [
              -0.8711,
              51.3682
            ],
            [
              -0.8711,
              51.3757
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T07:30:00.000+0000",
      "UUID": "81fa4a27-d21f-404b-b9bb-ba88bfe9c830",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.8711,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.3757,
      "longitude": -0.8672444204593983,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-18T21:14:00.000+0000",
      "latitude": 51.45890511319713,
      "estimatedRestoration": "2022-06-19T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0313'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0313",
      "reference": "JV0313",
      "maxLongitude": -0.5333,
      "minLatitude": 51.458,
      "name": "TW19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "TW19 5NJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5447,
              51.4691
            ],
            [
              -0.5333,
              51.4691
            ],
            [
              -0.5333,
              51.458
            ],
            [
              -0.5447,
              51.458
            ],
            [
              -0.5447,
              51.4691
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T08:00:00.000+0000",
      "UUID": "ecc4a0f3-8052-4a46-84f4-c57645815e99",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.5447,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4691,
      "longitude": -0.5413079822253984,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-18T20:11:00.000+0000",
      "latitude": 51.51339682070527,
      "estimatedRestoration": "2022-06-19T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0276'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0276",
      "reference": "JV0276",
      "maxLongitude": -0.5657,
      "engineerOnSiteTime": "2022-06-18T21:30:00.000+0000",
      "minLatitude": 51.5111,
      "name": "SL2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL1 3QE",
        "SL2 5BZ",
        "SL2 5XD",
        "SL2 5XE",
        "SL2 5XF",
        "SL2 5XH",
        "SL2 5XJ",
        "SL2 5XL",
        "SL2 5XN",
        "SL2 5XP",
        "SL2 5XW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5974,
              51.5157
            ],
            [
              -0.5657,
              51.5157
            ],
            [
              -0.5657,
              51.5111
            ],
            [
              -0.5974,
              51.5111
            ],
            [
              -0.5974,
              51.5157
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "a42ec58f-67ca-4b33-abad-2c042a61d584",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.5974,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5157,
      "longitude": -0.5726856419516447,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T04:07:00.000+0000",
      "latitude": 50.80168575675352,
      "estimatedRestoration": "2022-06-19T08:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0368'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0368",
      "reference": "JV0368",
      "maxLongitude": -1.1378,
      "engineerOnSiteTime": "2022-06-19T01:00:00.000+0000",
      "minLatitude": 50.8009,
      "name": "PO12 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO12 4QD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.14,
              50.8021
            ],
            [
              -1.1378,
              50.8021
            ],
            [
              -1.1378,
              50.8009
            ],
            [
              -1.14,
              50.8009
            ],
            [
              -1.14,
              50.8021
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T01:30:00.000+0000",
      "UUID": "4a2c33fe-adb8-4a42-a18c-32742c329283",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.14,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.8021,
      "longitude": -1.1392111585679021,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T03:01:00.000+0000",
      "latitude": 51.62223384810673,
      "estimatedRestoration": "2022-06-19T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0390'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0390",
      "reference": "JV0390",
      "maxLongitude": -1.983,
      "engineerOnSiteTime": "2022-06-19T03:47:00.000+0000",
      "minLatitude": 51.6191,
      "name": "SN16 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN16 9PY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0082,
              51.6385
            ],
            [
              -1.983,
              51.6385
            ],
            [
              -1.983,
              51.6191
            ],
            [
              -2.0082,
              51.6191
            ],
            [
              -2.0082,
              51.6385
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "225da269-d6d3-46de-9ac3-e73fa73221a2",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.0082,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.6385,
      "longitude": -1.9930909763023423,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T06:04:00.000+0000",
      "latitude": 51.91509367635237,
      "estimatedRestoration": "2022-06-19T08:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0400'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0400",
      "reference": "JV0400",
      "maxLongitude": -1.6017,
      "minLatitude": 51.9097,
      "name": "OX7 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX7 6FW",
        "OX7 6TA",
        "OX7 6YA",
        "OX7 6YD",
        "OX7 6YE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6222,
              51.9206
            ],
            [
              -1.6017,
              51.9206
            ],
            [
              -1.6017,
              51.9097
            ],
            [
              -1.6222,
              51.9097
            ],
            [
              -1.6222,
              51.9206
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T07:00:00.000+0000",
      "UUID": "a9acda30-225c-4d2b-bce9-3915b0b5c890",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.6222,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.9206,
      "longitude": -1.6215275194860816,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T05:11:00.000+0000",
      "latitude": 50.719929647538336,
      "estimatedRestoration": "2022-06-19T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0403'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0403",
      "reference": "JV0403",
      "maxLongitude": -1.2989,
      "engineerOnSiteTime": "2022-06-19T06:20:00.000+0000",
      "minLatitude": 50.7176,
      "name": "PO30 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO30 5TJ",
        "PO30 5TY",
        "PO30 5UF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3156,
              50.7234
            ],
            [
              -1.2989,
              50.7234
            ],
            [
              -1.2989,
              50.7176
            ],
            [
              -1.3156,
              50.7176
            ],
            [
              -1.3156,
              50.7234
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T06:45:00.000+0000",
      "UUID": "1b730a3b-f8e3-4a26-9eb5-3e67a6df6dc4",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3156,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7234,
      "longitude": -1.3061106191842426,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T06:21:00.000+0000",
      "latitude": 51.59735408745285,
      "estimatedRestoration": "2022-06-19T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0422'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0422",
      "reference": "JV0422",
      "maxLongitude": -1.3358,
      "engineerOnSiteTime": "2022-06-19T07:00:00.000+0000",
      "minLatitude": 51.5955,
      "name": "OX12 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX12 8JN",
        "OX12 8JR",
        "OX12 8JT",
        "OX12 8JW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3435,
              51.5996
            ],
            [
              -1.3358,
              51.5996
            ],
            [
              -1.3358,
              51.5955
            ],
            [
              -1.3435,
              51.5955
            ],
            [
              -1.3435,
              51.5996
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T07:00:00.000+0000",
      "UUID": "55e3bb15-1bc8-45fc-ab91-42cb0a5a3be6",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3435,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5996,
      "longitude": -1.3386938401173483,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T06:09:00.000+0000",
      "latitude": 50.712747764361765,
      "estimatedRestoration": "2022-06-19T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0419'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0419",
      "reference": "JV0419",
      "maxLongitude": -1.9791,
      "minLatitude": 50.7122,
      "name": "BH15 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH15 1QX",
        "BH15 1QY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9815,
              50.7131
            ],
            [
              -1.9791,
              50.7131
            ],
            [
              -1.9791,
              50.7122
            ],
            [
              -1.9815,
              50.7122
            ],
            [
              -1.9815,
              50.7131
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T07:15:00.000+0000",
      "UUID": "d97834ca-fd70-4a15-9d21-8475187f0011",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.9815,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7131,
      "longitude": -1.9801474634213763,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T06:52:00.000+0000",
      "latitude": 51.226004543222075,
      "estimatedRestoration": "2022-06-19T10:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0428'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0428",
      "reference": "JV0428",
      "maxLongitude": -1.4711,
      "minLatitude": 51.2252,
      "name": "SP10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP10 5JZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4742,
              51.2269
            ],
            [
              -1.4711,
              51.2269
            ],
            [
              -1.4711,
              51.2252
            ],
            [
              -1.4742,
              51.2252
            ],
            [
              -1.4742,
              51.2269
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T08:00:00.000+0000",
      "UUID": "8acc9498-8297-44fc-a762-6d844be8e0d0",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.4742,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.2269,
      "longitude": -1.473371424827635,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T05:48:00.000+0000",
      "latitude": 51.51258374553777,
      "estimatedRestoration": "2022-06-19T10:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0413'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0413",
      "reference": "JV0413",
      "maxLongitude": -0.3059,
      "minLatitude": 51.5118,
      "name": "W5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "W5 5AH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.308,
              51.513
            ],
            [
              -0.3059,
              51.513
            ],
            [
              -0.3059,
              51.5118
            ],
            [
              -0.308,
              51.5118
            ],
            [
              -0.308,
              51.513
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-06-19T07:45:00.000+0000",
      "UUID": "64fcca6f-65ec-4d9e-9b4d-8522e9a2cae2",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.308,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.513,
      "longitude": -0.30675002317975075,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-06-19T05:50:00.000+0000",
      "latitude": 50.725427877317095,
      "estimatedRestoration": "2022-06-19T10:20:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JV0414'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JV0414",
      "reference": "JV0414",
      "maxLongitude": -1.9307,
      "engineerOnSiteTime": "2022-06-19T06:40:00.000+0000",
      "minLatitude": 50.7248,
      "name": "BH14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH14",
        "BH14 9AW",
        "BH14 9HF",
        "BH14 9HG",
        "BH14 9HN",
        "BH14 9HW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9356,
              50.7259
            ],
            [
              -1.9307,
              50.7259
            ],
            [
              -1.9307,
              50.7248
            ],
            [
              -1.9356,
              50.7248
            ],
            [
              -1.9356,
              50.7259
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "fa735545-11d7-4ffa-9b88-ca3ecb76f296",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.9356,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7259,
      "longitude": -1.932979167550472,
      "resolved": false
    }
  ]
}
